pages/koneet-kiertoon.tsx

```diff
@@ -0,0 +1,411 @@
+import Head from 'next/head'
+import styled from 'styled-components'
+import { Content } from '../components/Content/Content'
+import { H1 } from '../components/Title/Title'
+import { Footer } from '../components/Footer/Footer'
+import { GetServerSideProps, InferGetServerSidePropsType } from 'next'
+import { Gallery } from '../components/Gallery/Gallery'
+import { InfoBox } from '../components/InfoBox/InfoBox'
+
+const Section = styled.div`
+
+  margin-top: 3rem;
+  margin-bottom: 1rem;
+
+  h2 {
+    margin: 0;
+    padding: 0;
+    font-family: 'Poppins';
+    font-weight: 600;
+    font-size: 1.8rem;
+  }
+
+  h3 {
+    margin: 0;
+    padding: 0;
+    font-family: 'Poppins';
+    font-weight: 600;
+    font-size: 1.rem;
+  }
+
+  h4 {
+    margin: 0;
+    padding: 0;
+    font-family: 'Poppins';
+    font-weight: 600;
+    font-size: 1rem;
+  }
+
+  p {
+    margin-bottom: 0;
+  }
+
+  svg {
+    vertical-align: middle;
+    margin-top: -3px;
+    opacity: 0.7;
+    margin-right: 0.3rem;
+  }
+`
+const Device = styled.div`
+
+  display: flex;
+  margin-top: 1rem;
+  flex-direction: row;
+  flex-wrap: wrap;
+  background-color: rgba(108, 108, 108, 0.09);
+  padding: 1rem;
```

Comment:
This `padding: 1rem;` should be removed because it's redefined on the line below.

Reply:
Fixed in [e4125c5](https://github.com/Testausserveri/testausserveri.fi/pull/99/commits/e4125c56cf8cab9d9ec0c55e15c9f1b8876d5c51)